docs/source/user-guide/getting-started.rst

```diff
@@ -2,306 +2,137 @@
-     snowflakes               /home/username/anaconda3/envs/snowflakes
-
-   The active environment is also displayed in front of your prompt in
-   (parentheses) or [brackets] like this:
+.. tip::
+   The active environment is the one with an asterisk (*).
 
-   .. code::
+To change your current environment back to the default ``base``::
 
-     (snakes) $
+   conda activate
 
-#. Verify which version of Python is in your current
-   environment:
+.. tip::
+    When the environment is deactivated, its name is no longer shown in your prompt, and the asterisk (*) returns to ``base``. To verify, you can repeat the  ``conda info --envs`` command.
 
-   .. code::
+Installing packages
+===================
 
-      python --version
+You can also install packages into a previously created environment. To do this, you first need to activate the environment. This changes the environment shown in your shell from ``(base)`` to the name of the environment. **Don’t install packages into your base environment.**::
 
-#. Deactivate the snakes environment and return to base environment:
-   ``conda activate``
+   conda activate myenvironment
+   conda install matplotlib
 
-   .. note::
-      For versions prior to conda 4.6, use:
+For more information on searching for and installing packages, see :doc:`Managing packages <tasks/manage-pkgs>`.
 
-        * Windows:  ``activate``
-        * macOS, Linux: ``source activate``
+Specifying channels
+===================
 
+Channels are locations (on your own computer or elsewhere on the Internet) where packages are stored. By default, conda searches for packages in its :ref:`default channels <default-channels>`.
 
-.. _managing-pkgs:
+If a package you want is somewhere else, such as conda-forge, you can manually specify the channel when installing the package::
 
-Managing packages
-=================
+   conda activate myenvironment
+   conda install conda-forge::numpy
 
-In this section, you check which packages you have installed,
-check which are available and look for a specific package and
-install it.
+You can also override the default channels in your `.condarc` file. For a direct example, see :ref:`Channel locations (channels) <config-channels>` or read the entire :doc:`Using the .condarc conda configuration file <configuration/use-condarc>`.
 
-#. To find a package you have already installed, first activate the environment
-   you want to search. Look above for the commands to
-   :ref:`activate your snakes environment <managing-envs>`.
+Updating conda
+==============
 
-#. Check to see if a package you have not installed named
-   "beautifulsoup4" is available from the Anaconda repository
-   (must be connected to the Internet):
+To see your conda version, use the following command::
 
-   .. code::
+   conda --version
 
-      conda search beautifulsoup4
+No matter where in you run this command, conda displays the number of the version that you have installed::
```

Comment:
```suggestion
No matter which environment you run this command in, conda displays its current version::
```

Is that what you meant?

Reply:
Woof, that was a bad sentence. Yeah, that's much better. Thanks!